packages/basics/src/tar-dump.js

```diff
@@ -1,7 +1,12 @@
+import { basename, resolve } from 'path';
+import { promisify } from 'util';
+import { readFile } from 'fs';
```

Comment:
Using `fs/promises` allows you to not use promisify.
See <https://nodejs.org/docs/latest-v20.x/api/fs.html#promises-api>

Reply:
absolutely right, that was my first choice, but it wasn't compatible with node12

https://github.com/Inist-CNRS/ezs/commit/aeec1dc3abd905647e429fc73f4ec48d7dd04ab7#diff-6b1bdd719a796e23f63a7a6ccc38f15df8db1d11702b6b30ebe2fd75013df674R2